### PR TITLE
I18n pull request restore with CrowdIn support

### PR DESCRIPTION
**Pull request by StanleyGoldman**
This PR restores parts of #1714 and adds initial integration with CrowdIn.

It is currently only providing translations in `zn-CH` for our `.resx` files. We're going to need a more scalable solution than duplicating the entirety of our `.vsct` files for every language so those files are going to be tackled in a separate PR.

The neutral language `en-US` resources are being stored directly in the assemblies themselves with satellite assemblies only being used for translations. This is because WPF does not work well with satellite assemblies for the neutral language.

### CODE REVIEW

**Comment by grokys** [comment]
Merged #1871 into this branch, but it appears that the `zh-HANS` translations aren't present here. Why is that?

**Comment by grokys** [comment]
@maikebing could take a look at this and tell me if I've got anything wrong?

File: test/GitHub.VisualStudio.UnitTests/Commands/OpenFromClipboardCommandTests.cs [hunk 1]
```diff
@@ -126,9 +146,13 @@ public async Task NoChangesInWorkingDirectory()
             vsServices.DidNotReceiveWithAnyArgs().ShowMessageBoxInfo(null);
             gitHubContextService.Received(1).TryOpenFile(repositoryDir, context);
         }
+        [Test]
```
Comment (jcansdale) [inline test/GitHub.VisualStudio.UnitTests/Commands/OpenFromClipboardCommandTests.cs]: Missing CR

File: test/GitHub.VisualStudio.UnitTests/Commands/OpenFromClipboardCommandTests.cs [hunk 2]
```diff
@@ -75,9 +79,25 @@ public async Task DifferentLocalRepository(string targetRepositoryName, string a
             }
         }
 
-        [TestCase("TargetOwner", "CurrentOwner", OpenFromClipboardCommand.NoResolveDifferentOwnerMessage)]
```
[the remark this answers is not in the document]
Comment (jcansdale) [reply test/GitHub.VisualStudio.UnitTests/Commands/OpenFromClipboardCommandTests.cs]: Something like this https://github.com/github/VisualStudio/pull/1888/commits/2baef978ccd1e4953bfbc63067be6ad86023354f.

**Comment by StanleyGoldman** [comment]
Where are the resources specified to be satellites?

**Comment by grokys** [comment]
@StanleyGoldman the  non-`en-US` .resx resources will be put in satellite assemblies.

**Pull request merged**